apps/module-scan/src/scanners/fujitsu.ts

```diff
@@ -72,7 +69,7 @@ export class FujitsuScanner implements Scanner {
       `--batch-prompt`,
     ]
 
-    if (this.pageSize === ScannerPageSize.Legal) {
+    if (pageSize === BallotPaperSize.Legal) {
```

Comment:
I recommend adding an `else if` for `Letter` that just says `// this is the default` or something and then an `else` that does `throwIllegalValue(pageSize)`. That way when we add additional page sizes there will be a compile error forcing us to handle it here.